examples/insert_file_stream_ndjson.ts

```diff
@@ -0,0 +1,49 @@
+import { createClient } from '@clickhouse/client'
+import Path from 'path'
+import Fs from 'fs'
+import split from 'split2'
+
+void (async () => {
+  const client = createClient()
+  const tableName = 'insert_file_stream_ndjson'
+  await client.exec({
+    query: `DROP TABLE IF EXISTS ${tableName}`,
+  })
+  await client.exec({
+    query: `
+      CREATE TABLE ${tableName} (id UInt64)
+      ENGINE MergeTree()
+      ORDER BY (id)
+    `,
+  })
+
+  // contains id as numbers in JSONCompactEachRow format ["0"]\n["0"]\n...
+  // see also: NDJSON format
+  const filename = Path.resolve(
+    process.cwd(),
+    './examples/resources/data.ndjson'
+  )
+
+  await client.insert({
+    table: tableName,
+    values: Fs.createReadStream(filename).pipe(
+      // for JSON* formats, it is currently required
+      // will be unnecessary when "insert" accepts a stream of strings/bytes
```

Comment:
The comment is outdated. `insert` accepts "raw" data, but we want to test `insert` of a stream of objects.